### PR TITLE
DOC: Added example for downloading a PDF from Google Cloud Storage 

### PR DESCRIPTION
**Pull request by prodeveloper**
Previously the documentation pointed to stackover flow which pointed to GCSFS. This project is still in beta.

This new example works with BytesIO meaning no need for another library and follows syntax similar to what was done for s3.

### CODE REVIEW

File: docs/user/streaming-data.md [hunk 1]
```diff
@@ -73,4 +73,19 @@ obj = s3.get_object(Body=csv_buffer.getvalue(), Bucket="my-bucket", Key="my/doc.
...
+To use with Google cloud storage
+
+```python
+from io import BytesIO
+from google.cloud import storage
+import os
+
+os.environ["GOOGLE_APPLICATION_CREDENTIALS"] = "Set this up either as secret or json file downloaded from gcloud"
+storage_client = storage.Client()
+blob = storage_client.bucket('my-bucket').blob('mydoc.pdf')
```
Comment (stefan6419846) [inline docs/user/streaming-data.md:85]:
```suggestion
blob = storage_client.bucket("my-bucket").blob("mydoc.pdf")
```

File: docs/user/streaming-data.md [hunk 2]
```diff
@@ -73,4 +73,19 @@ obj = s3.get_object(Body=csv_buffer.getvalue(), Bucket="my-bucket", Key="my/doc.
...
 
-It works similarly for Google Cloud Storage ([example](https://stackoverflow.com/a/68403628/562769)).
+To use with Google cloud storage
+
+```python
+from io import BytesIO
+from google.cloud import storage
+import os
+
+os.environ["GOOGLE_APPLICATION_CREDENTIALS"] = "Set this up either as secret or json file downloaded from gcloud"
```
Comment (stefan6419846) [inline docs/user/streaming-data.md:83]: Please add a comment instead which indicates that the user has to set this environment variable.

Comment (prodeveloper) [reply docs/user/streaming-data.md:83]: Ok will do thank you

File: docs/user/streaming-data.md (hunk 1 above)
Comment (prodeveloper) [reply docs/user/streaming-data.md:85]: Males sense will change. Thank you

File: docs/user/streaming-data.md [hunk 3]
```diff
@@ -73,4 +73,18 @@ obj = s3.get_object(Body=csv_buffer.getvalue(), Bucket="my-bucket", Key="my/doc.
 reader = PdfReader(BytesIO(obj["Body"].read()))
 ```
 
-It works similarly for Google Cloud Storage ([example](https://stackoverflow.com/a/68403628/562769)).
+To use with Google cloud storage
```
Comment (stefan6419846) [inline docs/user/streaming-data.md:76]:
```suggestion
To use with Google cloud storage:
```

File: docs/user/streaming-data.md [hunk 4]
```diff
@@ -73,4 +73,18 @@ obj = s3.get_object(Body=csv_buffer.getvalue(), Bucket="my-bucket", Key="my/doc.
...
+
+```python
+from io import BytesIO
+from google.cloud import storage
+import os
+# os.environ["GOOGLE_APPLICATION_CREDENTIALS"] must be set
```
Comment (stefan6419846) [inline docs/user/streaming-data.md:82]:
```suggestion

# os.environ["GOOGLE_APPLICATION_CREDENTIALS"] must be set 
```

File: docs/user/streaming-data.md [hunk 5]
```diff
@@ -73,4 +73,18 @@ obj = s3.get_object(Body=csv_buffer.getvalue(), Bucket="my-bucket", Key="my/doc.
...
+from io import BytesIO
+from google.cloud import storage
+import os
+# os.environ["GOOGLE_APPLICATION_CREDENTIALS"] must be set 
+storage_client = storage.Client()
+blob = storage_client.bucket('my-bucket').blob('mydoc.pdf')
+file_stream = BytesIO()
+blob.download_to_file(file_stream)
+reader = PdfReader(file_stream)
+
```
Comment (stefan6419846) [inline docs/user/streaming-data.md:88]:
```suggestion
```

File: docs/user/streaming-data.md [hunk 6]
```diff
@@ -73,4 +73,18 @@ obj = s3.get_object(Body=csv_buffer.getvalue(), Bucket="my-bucket", Key="my/doc.
 reader = PdfReader(BytesIO(obj["Body"].read()))
 ```
 
-It works similarly for Google Cloud Storage ([example](https://stackoverflow.com/a/68403628/562769)).
+To use with Google cloud storage:
```
Comment (stefan6419846) [inline docs/user/streaming-data.md:76]:
```suggestion
To use with Google Cloud storage:
```

**Review by stefan6419846 — DISMISSED** [review]

File: docs/user/streaming-data.md [hunk 7]
```diff
@@ -73,4 +73,18 @@ obj = s3.get_object(Body=csv_buffer.getvalue(), Bucket="my-bucket", Key="my/doc.
 reader = PdfReader(BytesIO(obj["Body"].read()))
 ```
 
-It works similarly for Google Cloud Storage ([example](https://stackoverflow.com/a/68403628/562769)).
+To use with Google Cloud storage:
+
+```python
+from io import BytesIO
+from google.cloud import storage
+import os
```
Comment (stefan6419846) [inline docs/user/streaming-data.md:81]:
```suggestion

```

File: docs/user/streaming-data.md [hunk 8]
```diff
@@ -73,4 +73,18 @@ obj = s3.get_object(Body=csv_buffer.getvalue(), Bucket="my-bucket", Key="my/doc.
 reader = PdfReader(BytesIO(obj["Body"].read()))
 ```
 
-It works similarly for Google Cloud Storage ([example](https://stackoverflow.com/a/68403628/562769)).
+To use with Google Cloud storage:
+
+```python
+from io import BytesIO
+from google.cloud import storage
```
Comment (stefan6419846) [inline docs/user/streaming-data.md:80]:
```suggestion

from google.cloud import storage
```

**Review by stefan6419846 — APPROVED** [review]

**Pull request merged**